editoast/justfile

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run:
+    cargo run
+
+# Install required dependencies
+install:
+    cargo build
+
+# Run the tests
+test:
+    cargo test -- --test-threads=4
+
+# Format the source code
+format:
+    cargo fmt
+
+# Verify code lints without applying any fix
+lint:
+    cargo clippy
+
+# Fix all the lints that can be automatically applied
+fix-lint:
+    cargo clippy --fix
```

(all-targets is probably activated by default, but better to match with the lint)
```suggestion
    cargo clippy --workspace --all-features --all-targets --fix
```